src/components/ReplyComment/index.js

```diff
@@ -129,7 +135,7 @@ const ReplyCommentId = ({
                           level={parseInt(level, 10) + 1}
                           updateVote={updateVote}
                           onLongPress={() => {
-                            showAlertDelete(itemReply, false, () => getThisComment(true));
+                            showAlertDelete(itemReply, false, updateComment);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure that `updateComment` is correctly implemented in `showAlertDelete`.
```diff
-                            showAlertDelete(itemReply, false, updateComment);
+                            showAlertDelete(itemReply, false, updateComment); // Ensure updateComment is correctly implemented in showAlertDelete
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->